task_0/vadim.marchenko/src/Main.java

```diff
@@ -0,0 +1,70 @@
+package src;
```

Comment:
Better not to name package as **src**
All the java code can be placed in src folder, but then there should go package such as **com.vadim**